aruodas/middlewares.py

```diff
@@ -0,0 +1,103 @@
+# Define here the models for your spider middleware
+#
+# See documentation in:
+# https://docs.scrapy.org/en/latest/topics/spider-middleware.html
+
+from scrapy import signals
+
+# useful for handling different item types with a single interface
+from itemadapter import is_item, ItemAdapter
+
+
+class AruodasSpiderMiddleware:
```

Comment:
Maybe (depends on the context) you can simply call this class `Spider` and below `Downloader`: it's more readable,  and if it must be imported later you can still do `from aruadas.middlewares import Spider as AruodasSpiderMiddleware` so that from the other file there will be no confusion